Magento_Captcha/templates/default.phtml

```diff
@@ -0,0 +1,50 @@
+<?php
+/** @var \Magento\Captcha\Block\Captcha\DefaultCaptcha $block */
+/** @var \Magento\Captcha\Model\DefaultModel $captcha */
+$captcha = $block->getCaptchaModel();
+?>
+<div role="<?= $block->escapeHtmlAttr($block->getFormId()) ?>">
```

Comment:
I'm curious about that `role` attribute. This is the custom one. I know that HTML5 validate `data-` attributes but `role ` attribute has some concrete values and when I did an accessibility audit of this page, it informed me that is not valid:
![screen shot 2018-03-27 at 17 51 29](https://user-images.githubusercontent.com/15080214/37978933-83ef0f34-31e7-11e8-9bfe-7fe19eb4a093.png)
could you please check it? If we can change it or is Magento issue? we shouldn't have invalid HTML ;)

Reply:
It's just a default magento error ;) I removed the entire div

Reply:
great 👍